[PATCH 1/2] Add /usr/sbin to the PATH of the condor_ce_config_generator cronjob (SOFTWARE-1643)
[PATCH 2/2] Run condor_ce_config_generator every 6 hours instead of every 10 minutes (SOFTWARE-1643)
---
 config/condor-ce-collector-generator.cron | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/config/condor-ce-collector-generator.cron
+++ b/config/condor-ce-collector-generator.cron
@@ -1 +1 @@
-*/10 * * * * root [[ ! -f /var/lock/subsys/condor-ce-collector ]] || /usr/bin/condor_ce_config_generator
+0 */6 * * * root [[ ! -f /var/lock/subsys/condor-ce-collector ]] || /usr/bin/env PATH="$PATH:/usr/sbin" /usr/bin/condor_ce_config_generator
-- 
2.0.1

